src/devices/sprut.ts

```diff
@@ -0,0 +1,112 @@
+import * as fz from "../converters/fromZigbee";
+import * as tz from "../converters/toZigbee";
+import * as m from "../lib/modernExtend";
+import * as reporting from "../lib/reporting";
+import type {DefinitionWithExtend} from "../lib/types";
+
+export const definitions: DefinitionWithExtend[] = [
+    {
+        zigbeeModel: ["Bed.box"],
+        model: "Bed.box",
+        vendor: "Sprut.device",
+        description: "Sprut bed.box Ergomotion controller",
+        fromZigbee: [fz.on_off],
+        toZigbee: [tz.on_off],
+        meta: {
+            multiEndpoint: true,
+        },
+        exposes: [],
+        extend: [
+            m.deviceEndpoints({
+                endpoints: {
+                    "1": 1,
+                    "2": 2, // headSet
+                    "3": 3, // legsSet
+                    "4": 4, // massageHeadIntensity 0: off, 1-6: intensity
+                    "5": 5, // massageLegsIntensity 0: off, 1-6: intensity
+                    "6": 6, // massageDuration 0: off, 1: 10 min, 2: 20 min, 3: 30 min
+                    flat: 7,
+                    zeroGravity: 8,
```

Comment:
```suggestion
                    zero_gravity: 8,
```

Please use `snake_case` here.